docker-compose.yaml

```diff
@@ -29,12 +29,19 @@ services:
       VULNHYDRATOR_HYDRATOR_NEXPOSE_PAGESIZE: ""
       VULNHYDRATOR_LAMBDAMODE: "false"
       VULNHYDRATOR_PRODUCERSIZELIMIT: 262144 # 256KiB in bytes
-  gateway:
+  gateway-incoming:
     build:
       context: .
-      dockerfile: gateway.Dockerfile
+      dockerfile: gateway-incoming.Dockerfile
     ports:
       - "8080:8080"
+  gateway-outgoing:
+    build:
+      context: .
+      dockerfile: gateway-outgoing.Dockerfile
+    ports:
+      - "8082:8082"
     environment:
       NEXPOSE_API_USERNAME:
       NEXPOSE_API_PASSWORD:
+      NEXPOSE_API_HOST:
```

Comment:
Task: you'll need to add a few more env vars here:
* `HTTPPRODUCER_API_HOST`
* `NEXPOSE_API_TIMEOUT`

Reply:
done!